trade-finance-manager-ui/server/controllers/case/cancellation/bank-request-date.controller.test.ts

```diff
@@ -1,8 +1,18 @@
 import { createMocks } from 'node-mocks-http';
+import { DayMonthYearInput } from '@ukef/dtfs2-common';
 import { aTfmSessionUser } from '../../../../test-helpers';
 import { PRIMARY_NAVIGATION_KEYS } from '../../../constants';
-import { BankRequestDateViewModel } from '../../../types/view-models';
-import { getBankRequestDate, GetBankRequestDateRequest } from './bank-request-date.controller';
+import { BankRequestDateValidationViewModel, BankRequestDateViewModel } from '../../../types/view-models';
+import { getBankRequestDate, GetBankRequestDateRequest, postBankRequestDate, PostBankRequestDateRequest } from './bank-request-date.controller';
+
+const validateBankRequestDateMock: jest.Mock<BankRequestDateValidationViewModel> = jest.fn(() => ({
+  errors: null,
+  bankRequestDate: undefined,
+}));
```

Comment:
Do we need to define `null` and `undefined` values here?

Reply:
I've changed it to: 

```
const validateBankRequestDateMock: jest.Mock<BankRequestDateValidationViewModel> = jest.fn(() => ({
  errors: null,
  bankRequestDate: new Date(),
}));
```

which is more realistic, as we expect either a date (and errors to be null) or errors and the bank request date to be undefined - thanks! Any errors we test are mocked specifically further down the file so fine for them to be null here.